src/PKSim.Core/Services/SimulationBuildingBlockUpdater.cs

```diff
@@ -140,7 +142,33 @@ public bool QuickUpdatePossibleFor(IPKSimBuildingBlock templateBuildingBlock, Us
          if (!BuildingBlockSupportsQuickUpdate(templateBuildingBlock))
             return false;
 
-         return templateBuildingBlock.StructureVersion == usedBuildingBlock.StructureVersion;
+         //not the same structure, easy return
+         var sameStructureVersion = templateBuildingBlock.StructureVersion == usedBuildingBlock.StructureVersion;
+         if (!sameStructureVersion)
+            return false;
+
+         //For individual, there is a special handling required as we need to also check that the used expression profile have hte same structure version
```

Comment:
Do we need a special handling for _populations_ as well?

Reply:
nope. We dont allow commit or update for population

Reply:
ok